client/src/components/NavBar/Notifications/NotificationCenter/NotificationCenter.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from 'react';
+import { Popover, IconButton } from '@material-ui/core';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import { User } from '../../../../interface/User';
+import NotificationItem from '../NotificationItem/NotificationItem';
+import INotificationItem from '../../../../interface/Notification';
+import { StyledBadge } from './notificationCenterStyles';
+
+type NotificationProps = {
+  loggedInUser: User;
+  notifications: INotificationItem[];
+};
+
+const NotificationCenter = ({ loggedInUser, notifications }: NotificationProps): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const [unread, setUnread] = useState(
+    notifications.filter((item: INotificationItem) => item.user === loggedInUser.email).length,
+  );
```

Comment:
I'm assuming this filter is just for testing right? And will be changed on integration

Reply:
Yes, that's the plan. We don't currently have any events set up to create notifications so this is just updating from the dummy data I created.